File: program/find-cube-root-of-a-number/FindCubeRootOfANumber.java
import java.util.Scanner;

public class FindCubeRootOfANumber {
    public static void main(String args[]){
        double i, precision = 0.000001;
        System.out.println("Enter a number:");
        Scanner sc = new Scanner(System.in);
        int num = sc.nextInt();

        for(i = 1; (i*i*i) <= num; ++i);
        for(--i; (i*i*i) < num; i += precision);
        System.out.println("Cube root of " + num + " is "+i);
    }
}

File: program/program/find-cube-root-of-a-number/find_cube_root_of_a_number.pl
print "Enter a number : \n";
$input = <STDIN>;
if($input<0){
$input*=-1; 
$out=-1*$input**(1/3);
$input*=-1; 
}
else{
$out=$input**(1/3);
}
&printout;

sub printout{
print "The cube root of $input is $out\n\n";
}
